webapp/packages/plugin-sql-editor/src/SqlEditorBootstrap.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { ConnectionsManagerService, IConnectionHandlerData } from '@cloudbeaver/core-connections';
+import { Bootstrap, injectable } from '@cloudbeaver/core-di';
+import { ExecutorInterrupter, IExecutionContextProvider } from '@cloudbeaver/core-executor';
+import { IServerConfigSaveData, ServerConfigurationService } from '@cloudbeaver/plugin-administration';
+import { AuthenticationService, LogoutState } from '@cloudbeaver/plugin-authentication';
+import { SqlEditorTabService } from '@cloudbeaver/plugin-sql-editor-navigation-tab';
+
+import { SqlEditorMenuService } from './SqlEditorMenuService';
+import { SqlResultTabsService } from './SqlResultTabs/SqlResultTabsService';
+
+@injectable()
+export class SqlEditorBootstrap extends Bootstrap {
+  constructor(
+    private readonly authenticationService: AuthenticationService,
+    private readonly sqlEditorTabService: SqlEditorTabService,
+    private readonly sqlResultTabsService: SqlResultTabsService,
+    private readonly sqlEditorMenuService: SqlEditorMenuService,
+    private readonly connectionsManagerService: ConnectionsManagerService,
+    private readonly serverConfigurationService: ServerConfigurationService
+  ) {
+    super();
+  }
+
+  register(): void {
+    this.sqlEditorMenuService.register();
+
+    this.authenticationService.onLogout.addHandler(this.logoutHandler.bind(this));
+    this.connectionsManagerService.onDisconnect.addHandler(this.connectionHandler.bind(this));
+    this.connectionsManagerService.onDelete.addHandler(this.connectionHandler.bind(this));
+    this.serverConfigurationService.saveTask.addHandler(this.saveHandler.bind(this));
```

Comment:
`serverConfigurationService` can't be used here
probably we don't want to check something from public side while we in administration

Reply:
also we can't use `sqlEditorTabService` and `sqlResultTabsService` here (because we don't want to know about tabs from `plugin-sql-editor`)